src/modules/Core/Account/settings.js

```diff
@@ -170,6 +192,13 @@ export const getCoreSettings = (account: AbcAccount): Promise<{otpMode: boolean,
   return Promise.resolve(coreSettings)
 }
 
+export async function getSyncedSettingsFileAsync (account: AbcAccount) {
```

Comment:
Why do you have this routine at all? It does the same thing as getSyncedSettingsFile and none of the operations inside are async in the first place

Reply:
I think I just got it mixed up with `getSyncedSettings`. I will remove it.